Basic/2DSpaceShooter/ProjectSettings/ProjectVersion.txt

```diff
@@ -1,2 +1,2 @@
-m_EditorVersion: 2020.3.7f1
-m_EditorVersionWithRevision: 2020.3.7f1 (dd97f2c94397)
+m_EditorVersion: 2020.3.12f1
+m_EditorVersionWithRevision: 2020.3.12f1 (b3b2c6512326)
```

Comment:
Could we match this with invaders? I think it is on 2020.3.13LTS

Reply:
2020.3.14f1 is the latest LTS currently. I was hoping it would be good enough if we have matching minors and ignore the weekly patches else maintenance will take a lot of time.

Reply:
Okay.